### ISSUE TITLE
AssertionError: Multiple normalization axes are not supported.

### ISSUE THREAD

**Issue by lygooi3** [Dobiasd/frugally-deep#383]
Hi,
I believe multiple normalization axes is not available in the latest version.
Not a new issue.
Is it possible to enable it?

My model is using 3 axes normalization on input image.

Error message:
File "/home/user/Documents/kerasC/frugally-deep/keras_export/convert_model.py", line 502, in show_normalization_layer
    assert len(layer.axis) <= 1, "Multiple normalization axes are not supported"
AssertionError: Multiple normalization axes are not supported

Affected layer:
layers.Normalization(axis=(1,2,3))

**Comment by Dobiasd** [comment]
Yeah, I did not implement multi-axes normalization, because I did not (yet) fully understand how it works (mathematically). :grimacing:

I'll see what I can do. :mechanic:

**Comment by lygooi3** [comment]
Hi,
Thank you for your quick response.
A mistake from my end, that leads to me looking for another new model to restart my project.

You may drop this request.

**Comment by Dobiasd** [comment]
Thanks for letting me know. :+1:

Maybe in the future I'll look into multi-axes normalization nonetheless, just out of curiosity. :slightly_smiling_face:
